jenkins/bootstrap.py

```diff
@@ -729,15 +729,20 @@ def job_script(job, use_json):
     """Return path to script for job."""
     if not use_json:
         return [test_infra('jobs/%s.sh' % job)]
+
     with open(test_infra('jobs/config.json')) as fp:
         config = json.loads(fp.read())
+    job_config = config.get(job)
+
+    if job_config is None:
+        with open(test_infra('jobs/generated/config.json')) as fp:
+            config = json.loads(fp.read())
```

Comment:
It seems to me that this would make it hard to modify the configs when regenerating them - users need to manually copy and paste the newly generated configs to replace the corresponding one in `config.json`. Is this still preferred than reading an extra file in `bootstrap.py`?